lib/pafs_core/date_utils.rb

```diff
@@ -25,5 +25,12 @@ def date_later_than?(date_name, other_date_name)
       Date.new(send("#{date_name}_year").to_i, send("#{date_name}_month").to_i, 1) >
         Date.new(send("#{other_date_name}_year").to_i, send("#{other_date_name}_month").to_i, 1)
     end
+
+    def date_in_future?(date_name)
+      send("#{date_name}_year").to_i > Time.zone.today.year || (
+        send("#{date_name}_year").to_i == Time.zone.today.year &&
+        send("#{date_name}_month").to_i > Time.zone.today.month
```

Comment:
Are dates in the current month not allowed?

Reply:
I think the current month should not be allowed. Because only year and month is provided, we treat the whole month as the current date.